practice_problems_3/fetch.php

```diff
@@ -0,0 +1,31 @@
+<?php
+if ($_SERVER["REQUEST_METHOD"] === "POST") {
```

Comment:
This will work for every post method for this page. Add a specific method with your post request.